### PR TITLE
Portfolio desktop version day1

### PR DESCRIPTION
**Pull request by Juanpa8830**
## Hi reviewer.
# These are the proposed changes for the project:

- finish portfolio desktop version.
- Add media queries in CSS file  to display on desktop screens.
- Configure  Grid and Flexbox.and replicate the design as much as possible with CSS. 🔢

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -120,67 +135,130 @@ <h3 class="tittle3">Finish mobile section 5</h3>
...
-        <section id="languages">
-            <h4>Languages</h4>
+<section id="projects-desktop"> 
 
-            <ul class="ul1">
-                <li>JavaScript(ES5/ES6)</li>
-                <li>Ruby</li>
-                <li>HTML</li>
-                <li>CSS</li>
+    <div class="div1"> <h2 class="deskt1">Projects</h2></div>
```
Comment (tufoinnkuo10) [inline index.html:148]: - [x] Great job. kindly align the below images well so they follow the figma design
yours
Screenshot (575)_LI

figma design
Screenshot (574)

File: index.html [hunk 2]
```diff
@@ -120,67 +135,130 @@ <h3 class="tittle3">Finish mobile section 5</h3>
...
+                        
                  <form class="form" action="https://formspree.io/f/xjvzpoww" method="post">
-                    <ul>
-                        <li><label for="text">Your name</label>
+
+                    <h1 class="GE">Get started</h1>
+
+                    <p class="firstpGE">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
+                    
+                    <ul id="savedata">
```
Comment (tufoinnkuo10) [inline index.html:259]: - [x] please ensure that the inputs are well aligned with the contact text so it looks like that of figma
Screenshot (576)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
## Hi @Juanpa8830 
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ design well followed


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 3]
```diff
@@ -17,25 +17,40 @@
...
     </header>
 
     <main>
         <section id="greetings">
-            <h1>Juan Pablo Guerrero Rios</h1>
-            <p class="firstp"> Hello! I am a software developer! I can help you build a product, feature or website.
+            
+            <div id="initial">
+                <h1>Juan Pablo Guerrero Rios</h1>
+                <p class="firstp"> Hello! I am a software developer! I can help you build a product, feature or website.
```
Comment (ridwanediallo) [inline index.html:35]: - [OPTIONAL ] Your paragraph `firstp` is moving all the way to the right. 
 
Screenshot from 2022-11-04 17-40-43

 Please I would highly recommend you fix it to make your project look more like the template.  What you can do is in your media query give it a                        
 `padding-left: 160px;`. 👌

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## @Juanpa8830 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Juanpa8830) [reply index.html:35]: Thank you, I already adjusted it!!

**Pull request merged**